demo/rmm-integration/rmm_bridge.cu

```diff
@@ -0,0 +1,35 @@
+#include <vector>
```

Comment:
There is no CUDA device code in this file or anything it includes, so I would make it a .cpp file so that it is compiled with the host compiler (NVCC is slow).

Reply:
I couldn't get it to work when I made it a cpp file. I got error `cudaFree not found` when I tried loading the library `librmm_bridge.so`.

Reply:
Hmm, was that a linker error? Because `cuda_memory_resource.hpp` has `#include <cuda_runtime_api.h>`. Perhaps you are missing `-lcudart`?

Reply:
I had the error when I tried loading the library.